.github/workflows/run-target-test-suite.yml

```diff
@@ -0,0 +1,85 @@
+# This workflow runs a specific regression test suite, using the input image
+# name and tag from the workflow call.
+#
+# Required settings in the repository:
+# Environments (exactly the following strings):
+# - production
+# - uat
+# - sit
+#
+# Secrets in each environment:
+# - EDL_USER
+# - EDL_PASSWORD
+#
+# Variables in each environment:
+# - HARMONY_HOST_URL
+#
+# Note: A workflow_call event can only have inputs of type boolean, number or
+# string.
+name: Run test suite
+
+on:
+  workflow_call:
+    inputs:
+      docker-image-name:
+        required: true
+        type: string
+      docker-image-tag:
+        required: false
+        type: string
+        default: 'latest'
+      harmony-environment:
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      docker-image-name:
+        required: true
+        type: choice
+        options:
+          - geoloco
+          - harmony-regression
+          - harmony
+          - hga
+          - hoss
+          - hybig
+          # - n2z  # Commented out because needs AWS credentials
```

Comment:
To make this work would just require someone to add an `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` secret to the repository, and then make sure the `Execute notebook` step in the workflow had those added as environment variables (like `EDL_USER`, etc). That would then entail someone to rotate these credentials to keep them up to date.